src/mlpack/methods/pca/pca_impl.hpp

```diff
@@ -61,7 +61,8 @@ void PCA<DecompositionPolicy>::Apply(const arma::mat& data,
 
 /**
  * Apply Principal Component Analysis to the provided data set.
- *
+ * It creates matrix to store eigenvectors and 
+   that matrix need not to be passed in paramteres
```

Comment:
I think that information is irrelevant to the user, since the interface doesn't expose the eigenvectors.